DOCS.md

```diff
@@ -93,9 +98,7 @@ publish:
     delete: true
     metadata:
       "*.png":
-        Cache-Control: "max-age: 10000000"
```

Comment:
These didn't properly set the `Cache-Control` header - just `x-amz-meta-cache-control`. See line 49 for the new example.